README.md

```
@@ -86,28 +106,56 @@ pnpm db:push     # Apply migrations to the database
 ### Email Sending
 - `POST /resend` - Resend the last sent email
 
+### Cron Jobs
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)